TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -166,7 +170,54 @@ public int GetTestCount()
             return _nbOfTests;
         }
 
-		public void Test(bool debug = false, bool json = false, bool autoRemarks = false) {
+#if DOMAIN_CHECKER
+        public static TypeCobolConfiguration DefaultConfig = null;
+        public static ProgramSymbolTableBuilder Builder = null;
+        public static NodeListenerFactory BuilderNodeListenerFactory = null;
+        public static string DefaultIntrinsicPath = null;//@"C:\TypeCobol\Sources\##Latest_Release##\Intrinsic\Intrinsic.txt";
+
+        public void TestInitialize()
+        {
+            SymbolTableBuilder.Root = null;
+            //Create a default configurations for options
+            DefaultConfig = new TypeCobolConfiguration();
+            if (File.Exists(DefaultIntrinsicPath))
+            {
+                DefaultConfig.Copies.Add(DefaultIntrinsicPath);
+            }
+            DefaultConfig.Dependencies.Add(Path.Combine(Directory.GetCurrentDirectory(), "resources", "dependencies"));
+            SymbolTableBuilder.Config = DefaultConfig;
+
+            //Force the creation of the Global Symbol Table
+            var global = SymbolTableBuilder.Root;
+
+            //Allocate a static Program Symbol Table Builder
+            BuilderNodeListenerFactory = () =>
```

Comment:
Is it possible to move this code (BuilderNodeListenerFactory and RegisterStaticNodeListenerFactory) into CompilationUnit ?
Because there a lot of class that need to do this manually.

The goal is to parse a (Type)Cobol program with the Parser API or CompilationProject/FileCompiler API. These APIs need to be simple to use.